bot/exts/events/hacktoberfest/_cog.py

```diff
@@ -0,0 +1,145 @@
+import logging
+import random
+from datetime import datetime
+
+from async_rediscache import RedisCache
+from discord.ext import commands
+
+import bot.exts.events.hacktoberfest._utils as utils
+from bot.constants import Client, Month
+from bot.utils.decorators import in_month
+from bot.utils.extensions import invoke_help_command
+
+log = logging.getLogger()
+
+
+class Hacktoberfest(commands.Cog):
+    """Cog containing all Hacktober-related commands."""
+
+    # Cache for `.hacktoberfest stats`which maps the discord user ID (as string) to their GitHub account
+    linked_accounts = RedisCache()
+
+    # Caches for `.hacktoberfest issue`
+    cache_normal = None
+    cache_timer_normal = datetime(1, 1, 1)
+    cache_beginner = None
+    cache_timer_beginner = datetime(1, 1, 1)
+
+    @commands.group(aliases=('hacktober',))
+    async def hacktoberfest(self, ctx: commands.Context) -> None:
+        """Commands related to Hacktoberfest."""
+        if not ctx.invoked_subcommand:
+            await invoke_help_command(ctx)
+            return
+
+    @in_month(Month.OCTOBER)
```

Comment:
```suggestion
    @in_month(Month.SEPTEMBER, Month.OCTOBER)
```
Perhaps people would like to start looking for issues beforehand if maintainers are already preparing for it